src/composition.rs

```diff
@@ -0,0 +1,157 @@
+// Copyright 2024 the Interpoli Authors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+use alloc::{string::String, vec::Vec};
+use core::ops::Range;
+use kurbo::{PathEl, Shape as _};
+
+#[cfg(feature = "std")]
+use std::collections::HashMap;
+
+use crate::{animated, Brush, Repeater, Stroke, Transform, Value};
+
+/// Model of a Lottie file.
+#[derive(Clone, Default, Debug)]
+pub struct Composition {
+    /// Frames in which the animation is active.
+    pub frames: Range<f64>,
+    /// Frames per second.
+    pub frame_rate: f64,
+    /// Width of the animation.
+    pub width: usize,
+    /// Height of the animation.
+    pub height: usize,
+    /// Precomposed layers that may be instanced.
+    #[cfg(feature = "std")]
+    pub assets: HashMap<String, Vec<Layer>>,
```

Comment:
I made this `#[cfg(feature = "std")]` to keep it compiling as it is to get it landed.

After it lands, I'll replace with `hashbrown` so that `no_std` works (with `alloc`).